### PR TITLE
DDCNL-2820 | removing categories from config

### PR DESCRIPTION
**Pull request by shikkhasingh**

### CODE REVIEW

File: app/views/paye/PayeGovernmentSpendingView.scala.html [hunk 1]
```diff
@@ -44,8 +44,8 @@ <h2 class="heading-xlarge flush--top">@Html(Messages("paye.generic.tax_year_head
 
 	<dl class="form-group" id="bar-chart">
 	@for(rowModel <- viewModel.orderedSpendRows){
-		<dt class="percentage" id="@rowModel.category">@Messages("paye.ats.treasury_spending.table." + rowModel.category) (@rowModel.spendData.percentage%)</dt>
-		<dd class="value"><span class="bar-chart" aria-hidden="true" style="width: @{rowModel.spendData.percentage*2}%;"></span> <strong>&pound;@rowModel.spendData.amount.toHalfRoundedUpAmount</strong></dd>
+		<dt class="percentage" id="@rowModel._1">@Messages("paye.ats.treasury_spending.table." + rowModel._1) (@rowModel._2.percentage%)</dt>
```
Comment (oc-ste) [inline app/views/paye/PayeGovernmentSpendingView.scala.html:47]: Is there a way to represent this data clearer? 

Perhaps wrapped up in a case class removing the need for `_1` and `_2`

File: test/view_models/paye/PayeGovernmentSpendSpec.scala [hunk 2]
```diff
@@ -57,12 +59,21 @@ class PayeGovernmentSpendSpec extends UnitSpec with MockitoSugar with JsonUtil w
...
+        result.orderedSpendRows.map(_._2.percentage) shouldBe PayeAtsTestData.expectedPercentageOrder
         result shouldBe PayeAtsTestData.payeGovernmentSpendViewModel.copy(
           isScottish = false
         )
       }
 
     }
+
+    "sort transport above Public Order" in {
+
```
Comment (oc-ste) [inline test/view_models/paye/PayeGovernmentSpendSpec.scala:71]: Is this tested in the backend? Not sure if you require a test here as it's basically whatever backend gives you?

Comment (shikkhasingh) [reply test/view_models/paye/PayeGovernmentSpendSpec.scala:71]: Backend is sorting the data based on percentage. But if percentage of transport and Public Order is same then we have to show transport above the public order so this reordering is done on frontend hence this test case.

File: app/views/paye/PayeGovernmentSpendingView.scala.html (hunk 1 above)
Comment (shikkhasingh) [reply app/views/paye/PayeGovernmentSpendingView.scala.html]: yes we can
removed _1 and _2 and wrapped up in a case class:)

**Review by oc-ste — APPROVED** [review]

**Pull request merged**